host_applications/linux/apps/raspicam/RaspiVid.c

```diff
@@ -141,10 +149,19 @@ typedef struct
    char  header_bytes[29];
    int  header_wptr;
    FILE *imv_file_handle;               /// File handle to write inline motion vectors to.
+   FILE *raw_file_handle;               /// File handle to write raw data to.
    int  flush_buffers;
    FILE *pts_file_handle;               /// File timestamps
 } PORT_USERDATA;
 
+/** Possible raw output formats
+ */
+typedef enum {
+   RAW_OUTPUT_FMT_YUV = 1,
+   RAW_OUTPUT_FMT_RGB,
+   RAW_OUTPUT_FMT_GRAY,
+} raw_output_fmt_t;
```

Comment:
Ok, will probably name it RAW_OUTPUT_FMT as other typedefs don't end with _T (PORT_USERDATA, RASPIVID_STATE).